build.sbt

```diff
@@ -1338,6 +1338,13 @@ lazy val `sbt-plugin` = (project in file("dev") / "sbt-plugin")
       val () = (publishLocal in `server-javadsl`).value
       val () = (publishLocal in `server-scaladsl`).value
       val () = (publishLocal in `spi`).value
+      val () = (publishLocal in `testkit-core`).value
+      val () = (publishLocal in `broker-javadsl`).value
+      val () = (publishLocal in `broker-scaladsl`).value
+      val () = (publishLocal in `pubsub-javadsl`).value
+      val () = (publishLocal in `pubsub-scaladsl`).value
+      val () = (publishLocal in `testkit-javadsl`).value
+      val () = (publishLocal in `testkit-scaladsl`).value
```

Comment:
The new `scripted` test requires `lagomJavadslTestKit` for a clustered app so a few more projects need publishing.

Reply:
How close are we of running a full `publishLocal`? I mean, should we remove all of those and just use `publishLocal` instead of `publishScriptedDependencies`?

Reply:
we were a handful short and #2242 brought those in. +1 to removing `publishScriptedDependencies`. Let's do it in a separate PR.

Reply:
We can use `.all(..)` with `ScopeFilter` to select all the projects so we don't have to maintain a only-just-smaller subset of projects.